types/tough-cookie-filestore/tough-cookie-filestore-tests.ts

```diff
@@ -0,0 +1,3 @@
+import FileCookieStore from 'tough-cookie-filestore';
```

Comment:
This should work if you enable `esModuleInterop` in the tsconfig.json file. Alternatively, use `import FileCookieStore = 'tough-cookie-filestore';` in this test, and just make sure you use `esModuleInterop` in your consuming code.